docs/usage/self-hosted-experimental.md

```diff
@@ -100,6 +100,10 @@ Allowed values:
 
 Default value: `asc`.
 
+## `RENOVATE_X_REBASE_PAGINATION_LINKS`
```

Comment:
@viceice I moved this PR into a draft state because I just noticed that this is impacting Renovate's calls to `github.com` to fetch release notes as well.  I'm looking into how to avoid that, and I see a few options:

1. Check that the base url of a pagination link not `https://api.github.com/` before rebasing it.
2. Change the environment variable to contain which base URL to remove.
3. Add an option in `GithubHttp.request(url, options)` to override the environment variable (ie, skip rebasing).

The first one avoids adding complexity to configuration and parameters.  What do you think?

Reply:
1. never rewrite for github.com

Reply:
added that in https://github.com/renovatebot/renovate/pull/19888/commits/c560a0ac5a3788e2be1cecb533787e89804e0318

Reply:
add a note that this only works for GitHub enterprise server

Reply:
added https://github.com/renovatebot/renovate/pull/19888/commits/f6b0376e32bf3bb658bb006ad36de66fc7a69b33

Reply:
please read our contribution guide and don't write such comments. re-request review instead.